src/components/FilterBar.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { makeStyles, Grid } from '@material-ui/core';
 import FilterDropDown from './FilterDropdown';
 import AmountFilter from './AmountFilter';
+import GradeFilter from './GradeLevelFilter';
```

Comment:
Naming in inconsistent here. Either update the grade filter import name or the file name

Reply:
ok I changed the import name